bucket/termshark.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "2.2.0",
+    "description": "A terminal UI for tshark, inspired by Wireshark",
+    "homepage": "https://github.com/gcla/termshark",
+    "license": "MIT",
+    "depends": "wireshark",
+    "notes": [
+        "Run 'scoop install wireshark' (from a elevated shell) to install 'tshark.exe' automatically"
+    ],
+
```

Comment:
So lets depend on manifest and then say to users this:

> Hey I just installed term shark and wireshark as dependency, but execute install command for wireshark manually to see a error

Reply:
@Ash258 could you suggest a better solution? 

sorry for late response